public/index.css

```diff
@@ -0,0 +1,75 @@
+div {
+  background-color: azure;
+  padding: 5px;
+}
+.container{
+  margin-top: 80px;
+}
+
+
+header {
+  position:fixed;
+  top:0;
+  width: 100%;
+  height: 62px;
+  color: #484848;
+  background-color: #fff;
+  border: 1px solid #DCE0E0;
```

Comment:
CSS is something that I was learning this weekend. I like the way you've managed to frame the 'new peep' and 'log in/log out' buttons at the top of the page. If I'm correct, then the 1px solid border in both the 'header' and 'main-nav li' are responsible for this. I presume there must be an easier way to create this format. Perhaps something to look into whenever you get a chance to play around with CSS a bit more.